README.rst

```diff
@@ -92,6 +92,20 @@ If you use PBxplore, please cite this tool as:
 | PBxplore: a tool to analyze local protein structure and deformability with Protein Blocks
 | *PeerJ*  5:e4013 `<https://doi.org/10.7717/peerj.4013>`_ (2017).
 
+The published version (1.3.8) is archived in 
```

Comment:
Add a new blank line above.
Currently, the "The plublished..." line is sticked to the reference one

Reply:
Thanks @HubLot 
Fixed!